lib/std/jsbigints.nim

```diff
@@ -209,3 +210,6 @@ runnableExamples:
     var b: JsBigInt
     doAssert b == big"0"
     doAssert b == JsBigInt.default
+  block `octal binary hexadecimal`:
```

Comment:
move this to the constructor `func big(integer: cstring): JsBigInt`, this example is getting too big otherwise

Reply:
I never seen users complaining of too much examples tho.